packages/core/src/Chip/ChipLabel/index.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+import { ChipContentWrapper } from '../ChipContentWrapper'
+import { getPaletteColor } from '../../utils'
+import { Label } from '../../Label'
+
+const ChipLabel = styled(Label)`
+  display: inline-flex;
+  width: auto;
+  user-select: none;
+  padding: 0;
+  margin: 0;
+  > input:focus ~ ${ChipContentWrapper} {
+    box-shadow: 0 0 0 1px ${getPaletteColor('base')};
```

Comment:
Are we able to use a `boxShadow` from theme, or is this custom? https://github.com/priceline/design-system/blob/master/packages/core/src/theme/theme.js#L123

cc @wesleymartin85

Reply:
@craigpalermo This is custom - Because outline can only have squared edges, this is a replacement, so it just looks like a focus outline with rounded corners

Reply:
We should consider adding to the theme though